lib/internal/Magento/Framework/Event/Invoker/InvokerDefault.php

```diff
@@ -39,15 +42,18 @@ class InvokerDefault implements \Magento\Framework\Event\InvokerInterface
     /**
      * @param \Magento\Framework\Event\ObserverFactory $observerFactory
      * @param State $appState
+     * @param \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
      * @param LoggerInterface $logger
      */
     public function __construct(
         \Magento\Framework\Event\ObserverFactory $observerFactory,
         State $appState,
+        \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig,
         LoggerInterface $logger = null
     ) {
         $this->_observerFactory = $observerFactory;
         $this->_appState = $appState;
+        $this->scopeConfig = $scopeConfig;
```

Comment:
Please use ObjectManager to initialize this

Reply:
Done